### ISSUE TITLE
Insufficient color contrast on Back to Top button

### ISSUE THREAD

**Issue by erinchambers** [CDRH/lewisandclark#251]
The new Back to Top button doesn't meet WCAG 2.1 color contast ratio **on hover**.
